include/hw/adsp/imx8m.h

```diff
@@ -41,6 +41,9 @@
 #define ADSP_IMX8M_DSP_MU_BASE       XSHAL_MU2_SIDEB_BYPASS_PADDR
 #define ADSP_IMX8M_DSP_MU_SIZE         0x10000
 
+#define ADSP_IMX8M_ESAI_BASE       0x59010000
+#define ADSP_IMX8M_ESAI_SIZE       0x00010000
+
 #define ADSP_IMX8M_SAI_3_BASE     0x30c30000
```

Comment:
@dianacretu there is no ESAI on i.MX8M as far as I know. So, lets remove this line.